programs/voter-stake-registry/src/lib.rs

```diff
@@ -234,17 +234,6 @@ impl Stake<'_> {
             MplStakingError::InvalidDelegate
         );
 
```

Comment:
This method should be renamed to `verify_delegate` as it's not touching the mining any more